Artifacts.toml

```diff
@@ -0,0 +1,6 @@
+[dash_textarea_autocomplete_resources]
+git-tree-sha1 = "<sha1>"
+
+    [[dash_textarea_autocomplete_resources.download]]
+    sha256 = "<sha256>"
+    url = "<url>"
```

Comment:
Feels like it might be a bit of a catch 22: we can't get the url until we do release, but we can't do a release without changing the value on L6.

We can maybe use a temporary resource like `DashTextareaAutoCompleteResources` ? 

Or maybe "we can't get the url until we do release" statement is wrong. That would be great.